src/kaleidoscope/plugin/Colormap.cpp

```diff
@@ -17,9 +17,6 @@
 
 
 #include <Kaleidoscope-Colormap.h>
-
-#include <EEPROM.h>
```

Comment:
No more required and causes problems with virtual builds.

Reply:
Maybe lift this to its own tiny PR we can merge in now?